web/beacon-app/src/features/auth/routes/RegistrationPage.tsx

```diff
@@ -20,7 +20,7 @@ export function Registration() {
       },
       {
         onSuccess: (_response) => {
-          localStorage.setItem('ensign.new.user', values.email);
+          localStorage.setItem('esg.new.user', values.email);
```

Comment:
This matches the key listed [here](https://github.com/rotationalio/ensign/blob/86bf18c668de6421014957e93e4f32ab61f10bc4/web/beacon-app/src/features/auth/components/Register/SuccessfullAccountCreation.tsx#L12).

Reply:
Nice find, I overlooked updating that value.